readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
```

Comment:
readme.md is conventionally named with this capitalization:

```README.md```

Reply:
Done